### PR TITLE
[e2e] External - Expand WPCOM suite, part 1

### PR DESCRIPTION
**Pull request by Stojdza**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Updates to the current `e2e-pw` setup, so tests are passing against the WPCOM environment. 

In this PR, we're covering `activate-and-setup`, `admin-analytics`, `admin-marketing` and `admin-tasks` to work with WPCOM. 

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. CI green - to be sure that changes in this PR are not affecting wp-env execution.
2. Locally - checkout this branch and run `pnpm test:e2e:with-env default-wpcom`. In `playwright.config.js` you can comment everything except this (to save some time for local execution):
```
'**/activate-and-setup/**/*.spec.js',
'**/admin-analytics/**/*.spec.js',
'**/admin-marketing/**/*.spec.js',
'**/admin-tasks/**/*.spec.js',
```
3. In the code, I put comments on why these changes.
4. `on-demand` job against this branch result is [here](https://a8c-woo-test-reports.s3.amazonaws.com/reports/other/on_demand_e2eexternalexpand-wpcom-suite-3191108/default-wpcom-core-e2e-and-api/index.html). Note: There are failures in "Store owner can skip the core profiler" which I skipped after in [df54612](https://github.com/woocommerce/woocommerce/pull/51303/commits/df5461240719fdbd10b5e8d582ac9fb5e87eda2f). 

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

Expand the e2e suite we're running on WPCOM. 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/tests/admin-marketing/overview.spec.js [hunk 1]
```diff
@@ -15,40 +15,44 @@ test.describe( 'Marketing page', () => {
...
 	} );
 
-	test( 'Marketing Overview page have relevant content', async ( {
-		page,
-	} ) => {
-		// Go to the Marketing page.
-		await page.goto( 'wp-admin/admin.php?page=wc-admin&path=%2Fmarketing' );
+	test(
+		'Marketing Overview page have relevant content',
+		{ tag: '@skip-on-default-wpcom' },
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/admin-marketing/overview.spec.js:20]: There's no "Discover more marketing tools" and related content on the WPCOM site.

File: plugins/woocommerce/tests/e2e-pw/tests/admin-tasks/payment.spec.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ test.describe( 'Payment setup task', () => {
 		await page
 			.locator( '//input[@placeholder="BIC / Swift"]' )
 			.fill( 'ABBA' );
-		await page.locator( 'text=Save' ).click();
+		await page.getByRole( 'button', { name: 'Save' } ).click();
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/admin-tasks/payment.spec.js:76]: `page.locator( 'text=Save' ).click();` was resolved to two elements on WPCOM. Make it more unique

**Review by adimoldovan — CHANGES_REQUESTED** [review]
@Stojdza I cannot get this one test to pass locally. 

`admin-tasks/payment.spec.js:92 > Can visit the payment setup task from the homescreen if the setup wizard has been skipped`

```
Error: locator.click: Error: strict mode violation: locator('text=Get paid') resolved to 3 elements:
    1) <h1>It’s time to get paid</h1> aka getByRole('heading', { name: 'It’s time to get paid' })
    2) <button type="button" class="components-button is-primary">Get paid</button> aka getByRole('button', { name: 'Get paid', exact: true })
    3) <span class="woocommerce-task-list__item-title">Get paid</span> aka getByRole('button', { name: '3 Get paid' })

Call log:
  - waiting for locator('text=Get paid')


  94 | 	} ) => {
  95 | 		await page.goto( 'wp-admin/admin.php?page=wc-admin' );
> 96 | 		await page.locator( 'text=Get paid' ).click();
     | 		                                      ^
  97 | 		await expect(
  98 | 			page.locator( '.woocommerce-layout__header-wrapper > h1' )
  99 | 		).toHaveText( 'Get paid' );

    at /plugins/woocommerce/tests/e2e-pw/tests/admin-tasks/payment.spec.js:96:41
```

**Comment by Stojdza** [comment]
Good finding @adimoldovan 👍 It was passing for me locally and on [CI as well](https://a8c-woo-test-reports.s3.amazonaws.com/reports/other/on_demand_e2eexternalexpand-wpcom-suite-3191108/default-wpcom-core-e2e-and-api/index.html#suites/a1c6540231e3830f3ea2dcd20446e572/b640e9a9277bcf35/). I'm not sure what has changed. 

Anyway, I followed Playwright "Pick locator" recommendation, and made `Get paid` button more unique - [ae8b193](https://github.com/woocommerce/woocommerce/pull/51303/commits/ae8b1934f6b37ab1f81e8af9def8aa90c7dfad6d). The test is passing against WPCOM now. I also ran it 50 times on wp-env and it passed every time.

**Review by adimoldovan — APPROVED** [review]
Tested well locally after the locator fix. All tests passed.

**Pull request merged**